highfidelity/hifi_tests
# 20339 ensure no entities

https://highfidelity.fogbugz.com/f/cases/20339/
# Description
Adds a script to delete all entities around the avatar.  This solution was chosen over hiding and the re-showing the entities for 2 reasons:
1.  There may be invisible entities - these then would become visible after the test
2.  This complicates nitpick even more.
# Testing
1.  Run Interface and go to localhost.
2.  Move to a remote region (such as 9999,9999,9999)
3.  Create some entities at that location - (i.e. cube, sphere and text)
4.  Run the following test: https://raw.githubusercontent.com/NissimHadar/hifi_tests/20339-ensureNoEntities/tests/utils/deleteNearbyEntities.js
5.  Verify the added entities have been deleted.  There should be no entities nearby after the test has been completed.
